app/src/main/java/com/sun/note_01/screen/login/fragment/childregister/RegisterChildFragment.kt

```diff
@@ -0,0 +1,17 @@
+package com.sun.note_01.screen.login.fragment.childregister
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.sun.note_01.R
+
+class RegisterChildFragment : Fragment() {
+    override fun onCreateView(
```

Comment:
func thường cách ở trên một dòng 
```suggestion

    override fun onCreateView(
```